dmscripts/export_framework_results_reasons.py

```diff
@@ -16,10 +16,17 @@
 ]
 
 
+def get_error_key(error):
+    try:
+        return error.path[0]
+    except (AttributeError, TypeError, IndexError):
+        raise ValueError(f"Unexpected validation error: {error!r}")
```

Comment:
Nice - tested locally with a missing `required` answer:

`ValueError: Unexpected validation error: <ValidationError: "'GAAR' is a required property">` 👍